packages/fxa-payments-server/package.json

```diff
@@ -5,13 +5,14 @@
   "scripts": {
     "lint": "npm-run-all --parallel lint:*",
     "lint:eslint": "eslint .",
-    "lint:tsc": "tsc --noEmit",
+    "lint:sass": "sass-lint -v",
     "start": "node server/bin/fxa-payments-server.js",
     "start-dev": "concurrently -k \"PROXY_STATIC_RESOURCES_FROM='http://127.0.0.1:3032' node server/bin/fxa-payments-server.js\" \"react-scripts start\"",
     "build": "react-scripts build",
     "eject": "react-scripts eject",
-    "test": "npm-run-all test:*",
-    "test:ts": "cross-env TS_NODE_COMPILER_OPTIONS='{\"module\":\"commonjs\", \"jsx\": \"react\"}' nyc --reporter=html --reporter=text mocha -r ts-node/register -r @babel/register -r jsdom-global/register -r test-setup.ts src/**/*-test.{ts,tsx} --recursive --timeout 5000"
+    "test": "react-scripts test",
```

Comment:
New / modified npm scripts, here:

* `npm run lint` - now also includes sass-lint
* `npm run test` - runs tests interactively, unless env var `CI=true`
* `npm run storybook` - runs a storybook web app at port 6006
* `npm run build-storybook` - builds the storybook app as a static website under './storybook` - may be useful later for [auto-publishing snapshots of the site](https://medium.com/firefox-test-pilot/storybook-test-pilot-26d21fdb6c64)

Reply:
Could you do a runthrough on how to use storybook? Looks interesting.